src/resolvers/ArticleResolver.ts

```diff
@@ -111,7 +111,7 @@ class ArticleResolver {
     const presentDate = new Date().getTime();
     // Due to the way Mongo interprets 'article' object,
     // article['_doc'] must be used to access fields of a article object
-    return article['_doc'].shoutouts / (presentDate - article['_doc'].date.getTime()); // eslint-disable-line
+    return article['_doc'].shoutouts / ((presentDate - article['_doc'].date.getTime())/1000); // eslint-disable-line
```

Comment:
this seems unrelated to this PR, why did we have to divide this by 1000

Reply:
The trendiness value was very small otherwise, as the dates in the denominator were in milliseconds

Reply:
It was to keep it consistent with the magazine trendiness formula